sdk/eventhubs/azure-messaging-eventhubs/samples/README.md

```diff
@@ -0,0 +1,35 @@
+# Eventhub Samples
+
+This repository contains samples for the Azure Event Hubs service.
+
+## Sample Requirements
+
+These samples are written with the assumption that the following environment
+variables have been set by the user:
+
+* EVENTHUBS_CONNECTION_STRING - The service connection string for the eventhubs instance.
+* EVENTHUB_NAME - Name of the eventhubs instance to communicate with.
+* EVENTHUBS_HOST - Fully qualified domain name for the eventhubs instance.
```

Comment:
Two of these is `Eventhubs` (plural), but one isn't. Should it be consistent?

Reply:
Two of these refer to the EventHubs service (which is plural), one of these refers to a specific instance of an eventhub (which by definition is singular).

IMHO, it may not be consistent, but it *is* correct.